lib/house.rb

```diff
@@ -0,0 +1,58 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(value, address)
+    @price = (value.sub("$", "")).to_i
```

Comment:
Nice job putting this conversion in initialize. This helps to avoid duplicating the work later on.